### DISCUSSION TITLE
CLI test outputs visibility?

### DISCUSSION THREAD

**Discussion by liketoeatcheese** [usebruno/bruno#2694] [Ideas]
Hi,

I have been using the test CLIs, and they look pretty good and have quite a potential. Is it possible to enable a flag to view the input and output of what's actually being sent and returned?

Furthermore, another flag to test like normal, but if things fail, it shows the details of what's actually being sent and failed.

This would be great since it helps in debugging or implementing CI/CD pipeline

**Comment by eifcgn24** [comment]
Hi, do you mean the "Timeline"?
https://stackoverflow.com/questions/77848215/is-there-a-way-to-see-the-requests-made-using-bruno

**Reply by liketoeatcheese** [reply]
Hi, yes. That's the one. It would be super helpful if those details could be printed to the stdout based on some type of flags. But without the ">," it's easier to highlight and copy.
